pulse_otel/main.py

```diff
@@ -456,6 +456,19 @@ def shutdown(self):
         # No specific shutdown logic needed for file-based exporting
         pass
 
+class JSONFileLogger:
```

Comment:
Should it just be a function? Or we need references to the created logger_provider or exporter?